pages/common/ssh-keygen.md

```diff
@@ -22,6 +22,10 @@
 
 `ssh-keygen -l -F {{remote_host}}`
 
+- Retrieve the fingerprint of a key in MD5 Hex (useful for checking a key against the list in Github):
```

Comment:
The page itself is getting pretty long too, btw. Should we consider dropping one of the current examples to add this one (if we agree the tradeoff is worth it)?